tests/rest/client/v2_alpha/test_register.py

```diff
@@ -130,3 +134,21 @@ def test_POST_disabled_guest_registration(self):
 
         self.assertEquals(channel.result["code"], b"403", channel.result)
         self.assertEquals(channel.json_body["error"], "Guest access is disabled")
+
+    def test_POST_ratelimiting(self):
+        for i in range(0,6):
+            request, channel = self.make_request(b"POST", self.url + b"?kind=guest", b"{}")
+            self.render(request)
+
+            if i is 5:
```

Comment:
You want `i == 5`, `is` is a bit special and while it works in this case its good not to get into the habit of using `is`.

Reply:
Roger that (I'm still learning how to do proper python)

Reply:
`is` tests for "identity" i.e. if the objects are literally the same object, whereas `==` tests for "equality"

Reply:
Oooh now I see how stuff could go wrong here.